lib/Reconciler.lua

```diff
@@ -37,7 +37,7 @@ local function isPortal(element)
 		return false
 	end
 
-	return element.type == Core.Portal
+	return element.component == Core.Portal
```

Comment:
Eurgh, I can't wait to rewrite the reconciler + tests so that we have better coverage for refactors like this...